cmake/FindNUMPY.cmake

```diff
@@ -39,11 +39,7 @@
 #============================================================================
 
 # Finding NumPy involves calling the Python interpreter
-if(NumPy_FIND_REQUIRED)
-    find_package(PythonInterp REQUIRED)
-else()
-    find_package(PythonInterp)
-endif()
+find_package(PythonInterp REQUIRED)
```

Comment:
Why did you change this? If FindNumPy is now called without REQUIRED, it will fail if there is no interpreter, right?